comp/swerve100/src/main/java/org/team100/frc2024/LEDSubsystem.java

```diff
@@ -4,9 +4,12 @@
 
 import org.team100.frc2024.motion.shooter.Shooter;
 import org.team100.lib.indicator.LEDIndicator;
+import org.team100.lib.indicator.LEDStrip;
 import org.team100.lib.indicator.LEDIndicator.State;
 import org.team100.lib.localization.VisionDataProvider24;
 
+import com.ctre.phoenix.led.CANdle.LEDStripType;
```

Comment:
i think you're using the wrong strip type here, you want something like LEDStrip.strip, i.e. the enum you made below.